### PR TITLE
Added new header

### PR DESCRIPTION
**Pull request by chomweaverNOAA**

### CODE REVIEW

**Review by stefaniebutland — APPROVED** [review]
Thanks for making a pull request @chomweaverNOAA! Ready for you to merge it.

File: github-clinic/cory.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Working on GitHub.com, we contribute changes through **commits**. You'll practic
 
 Let's edit this file by clicking the pencil icon at the top-right of the file. 
 
-Next, make an edit to the text in this file. One idea is to fix this tpyo.
```
Comment (brooke-l-hawkins-noaa) [inline github-clinic/cory.md:9]: Trying out a line by line comment!

File: github-clinic/cory.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ We can make words **bold** or *italic*.
 
 ### We can make headers.
 
+### **New header**!
```
Comment (brooke-l-hawkins-noaa) [inline github-clinic/cory.md:25]:
```suggestion
#### **New header**!
```

**Review by brooke-l-hawkins-noaa — APPROVED** [review]
Test finishing a review

**Pull request merged**